server/src/main/java/com/focusit/jsflight/server/repository/EventRepository.java

```diff
@@ -9,7 +9,7 @@
 /**
  * Created by dkirpichenkov on 20.05.16.
  */
-public interface EventRepositoryCustom
+public interface EventRepository
```

Comment:
Do we really need this postfix? I mean, that it's not so hard to look into implementation. But without postfix it look better, more consistently